src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
The class should have an access modifier, preferably 'public' to ensure proper visibility.